lib/cc/analyzer/formatters/html_formatter.rb

```diff
@@ -1,34 +1,280 @@
 require "redcarpet"
-require "active_support/number_helper"
 
 module CC
   module Analyzer
     module Formatters
-      class HTMLFormatter < Formatter
+      class HTMLFormatter < Formatter # rubocop: disable Metrics/ClassLength
+        LANGUAGES = Hash.new { |_, ext| ext }.
+                    merge(
+                      # abap
```

Comment:
The commented out are the ones that map to themselves.

The default is to use file extension as the syntax used to highlight that file. So entry like `'abap' => 'abap'` is redundant. But I left comments for the ones that are actually supported by Prism.

This list is completely based on what's supported by Prism, not what CC can actually  analyze.